extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -595,7 +655,47 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
               visiblePercentageMin: 50,
               continuousTimeMin: 1,
             });
-    return superReturnValue;
+    if (this.isFluid_ && !this.element.style.width) {
```

Comment:
What is the pub specifies a width via CSS?  Wouldn't you be overriding it?

Reply:
Wouldn't the `&& !this.element.style.width` catch that?

Reply:
Oh, I see what you mean.

Reply:
Well, as far as I see, the only alternative is to call computedStyle then. Not sure which is preferable.